core-service/src/main/java/controllers/ParkingSlotController.java

```diff
@@ -0,0 +1,8 @@
+package controllers;
```

Comment:
* Remove this package and create separate repo for core service.
* Also change repo name to **ParkingTicketSystem-CommonEntities**.

Reply:
My focus is on making code workable not different repo. 
We can do if it is important at any import that is why I create the different module for the different component.